### PR TITLE
UUID CRUD operations

### LINKED ISSUE #1355: Add support for UUID
* View
* Edit

### PR DESCRIPTION
**Pull request by steffenagger**
### Info:
UUID CRUD operations both for single instances & lists.

Simple file for testing: [test-realm-file.zip](https://github.com/realm/realm-studio/files/6156698/test-realm-file.zip)

This PR also contains a shift from the "external" `'bson'` package as a depencency, and consumes only what comes through the `'realm'` package.

And a minor UI fix in relation to spell checking in cells.

### Warning:
~~This PR utilized realm from a dev GH branch (`github:realm/realm-js#datatype/uuid`), and should not be merge in its current state.~~ Updated to latest alpha.

### Affects:
closes: #1355

### CODE REVIEW

File: src/ui/RealmBrowser/Content/Table/types/StringCell/StringCell.tsx
Comment (steffenagger) [inline src/ui/RealmBrowser/Content/Table/types/StringCell/StringCell.tsx:54]: Not sure if we should disable it altogether (a red squiggly line can be missread as a cells error-state). For now left in for 'string' types.

File: src/ui/RealmBrowser/Content/Table/types/ListCell.tsx
Comment (kraenhansen) [inline src/ui/RealmBrowser/Content/Table/types/ListCell.tsx:71]: I think we need to locate the root cause of this.

**Review by kraenhansen — APPROVED** [review]
Let's wait for an official release of Realm JS before merging this.

File: src/ui/RealmBrowser/Content/Table/types/ListCell.tsx
Comment (steffenagger) [reply src/ui/RealmBrowser/Content/Table/types/ListCell.tsx:71]: 100% agreed

Comment (steffenagger) [reply src/ui/RealmBrowser/Content/Table/types/ListCell.tsx]: Ok I investigated, and - I goofed...
This was fixed by other edits in this PR (it was not yet detecting a `Realm.List<UUID>` as primitives... and treated them as a list of `Realm.Object`).

So this is a non-issue.
I rolled back the quickfix.

**Pull request merged**
